manual/dmc.tex

```diff
@@ -37,7 +37,7 @@ \section{Diffusion Monte Carlo}
    &   \texttt{reconfiguration     } &  string  & yes/pure/other & no   & fixed population technique  \\
    &   \texttt{branchInterval      } &  integer  & $\ge 0$ & 1   & branching interval \\
    &   \texttt{substeps            } &  integer  & $\ge 0$ & 1   & branching interval \\
-   &   \texttt{nonlocalmoves       } &  string  & yes/v0/v1/no & no   & run with tmoves  \\
+   &   \texttt{nonlocalmoves       } &  string  & see below & no   & run with tmoves  \\
```

Comment:
Please update with full list of options. yes/v0/v1/v2/no

Reply:
Unfortunately, that row goes too wide. I prefer not to list all options.

Reply:
TeX harder. This is a common option. We can have two lines, smaller fonts...